validator/src/bin/solana-test-validator.rs

```diff
@@ -204,7 +204,7 @@ fn main() {
 
                     programs_to_load.push(ProgramInfo {
                         program_id: address,
-                        loader: solana_sdk::bpf_loader_upgradeable::id(),
+                        loader: solana_sdk::bpf_loader::id(),
```

Comment:
Hey,

I had the same idea, But in genesis we pass the bootloader program id to specify which loader to use for the program.
Is old loader deprecated and we now use this loader for all the programs?

Reply:
I don't understand your question. This change is a revert *to* the old immutable loader.
We are, however, considering proper support for the upgradeable loader in solana-test-validator, as well as defaulting to the upgradeable loader: https://github.com/solana-labs/solana/pull/30412